Libplanet.Tests/Common/Action/ThrowException.cs

```diff
@@ -13,14 +12,15 @@ public ThrowException()
         {
         }
 
-        public bool ThrowOnRehearsal { get; set; } = false;
+        public bool ThrowOnRehearsal { get; set; }
 
-        public bool ThrowOnExecution { get; set; } = false;
+        public bool ThrowOnExecution { get; set; }
```

Comment:
Removing these values does affect nothing, right?

Reply:
Yeah, this does affect nothing.